superdesk/users/services.py

```diff
@@ -411,3 +413,62 @@ def on_deleted(self, doc):
 
         super().on_deleted(doc)
         get_resource_service('reset_user_password').remove_all_tokens_for_email(doc.get('email'))
+
+
+class UserSessionClearResource(Resource):
+    endpoint_name = 'clear_sessions'
+    url = 'users/<{0}:user_id>/clear_sessions'.format(item_url)
```

Comment:
Can we have URL like this `DELETE users/<user_id>/sessions` instead of `POST DELETE users/<user_id>/sessions`?